google/compute/v1/compute_gapic.yaml

```diff
@@ -0,0 +1,101 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.compute.spi.v1
+  python:
+    package_name: google.cloud.gapic.compute.v1
+  go:
+    package_name: cloud.google.com/go/compute/apiv1
+  csharp:
+    package_name: Google.Compute.V1
+  ruby:
+    package_name: Google::Cloud::Compute::V1
+  php:
+    package_name: Google\Cloud\Compute\V1
+  nodejs:
+    package_name: compute.v1
+license_header:
+  copyright_file: copyright-google.txt
+  license_file: license-header-apache-2.0.txt
+interfaces:
+- name: google.compute.v1.InstancesService
+  collections: []
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 20000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 20000
+    total_timeout_millis: 600000
+  methods:
+  - name: ListInstances
+    required_fields:
+    - max_results
+    - filter
+    - order_by
+    - project
+    - zone
+    request_object_method: true
+    page_streaming:
+      request:
+        token_field: page_token
+      response:
+        token_field: next_page_token
+        resources_field: items
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: GetInstance
+    flattening:
+      groups:
+      - parameters:
+        - project
+        - zone
+        - instance
+    required_fields:
+    - project
+    - zone
+    - instance
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: InsertInstance
+    flattening:
+      groups:
+      - parameters:
+        - project
+        - zone
+        - resource
```

Comment:
(This comment thread would probably be better on the proto)

The discovery doc doesn't have a name for this field because it is just set as the post body. The original proto2 called it `resource`. Maybe `instance` is a better name as you say.